### PR TITLE
CCloudify aggregate sum

### PR DESCRIPTION
**Pull request by awalther28**
### Description

CCloudify aggregate sum KT. Add short answer for kstreams and confluent. 

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/convert-stateful-agg-sum/create-stateful-aggregation-sum/confluent.html

### New tutorial checklist

- [x] Add a `Short Answer`
- [ ] Implement good test cases
- [ ] Validate hyperlinks
- [ ] Spell check (e.g. using `aspell`)

### CODE REVIEW

File: _includes/tutorials/aggregating-sum/kstreams/code/src/main/java/io/confluent/developer/AggregatingSum.java [hunk 1]
```diff
@@ -64,37 +53,35 @@ public Topology buildTopology(Properties envProps,
         // Apply SUM aggregation
         .reduce(Integer::sum)
         // Write to stream specified by outputTopic
-        .toStream().to(outputTopic, Produced.with(Serdes.String(), Serdes.Integer()));
+        .toStream().mapValues(v -> v.toString()).to(outputTopic, Produced.with(Serdes.String(), Serdes.String()));
```
Comment (bbejeck) [inline _includes/tutorials/aggregating-sum/kstreams/code/src/main/java/io/confluent/developer/AggregatingSum.java:56]: one nit change to `(v -> v + " total sales")`

**Review by bbejeck — COMMENTED** [review]
Thanks for the PR @awalther28 just one minor nit otherwise this looks good

File: _includes/tutorials/aggregating-sum/kstreams/code/src/main/java/io/confluent/developer/AggregatingSum.java (hunk 1 above)
Comment (awalther28) [reply _includes/tutorials/aggregating-sum/kstreams/code/src/main/java/io/confluent/developer/AggregatingSum.java]: Thank you for the review and comment @bbejeck I updated the code/tests/output with your suggestion in commit https://github.com/confluentinc/kafka-tutorials/pull/814/commits/8bbc18c43deb6632d79bcf246209cfcfffa37f0b

File: _includes/tutorials/aggregating-sum/kstreams/markup/answer/short-answer.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+```
+    builder.stream(inputTopic, Consumed.with(Serdes.String(), ticketSaleSerde))
+        // Set key to title and value to ticket value
+        .map((k, v) -> new KeyValue<>((String) v.getTitle(), v.getTicketTotalValue()))
+        // Group by title
+        .groupByKey(Grouped.with(Serdes.String(), Serdes.Integer()))
+        // Apply SUM aggregation
+        .reduce(Integer::sum)
+        // Write to stream specified by outputTopic
+        .toStream().mapValues(v -> v.toString()).to(outputTopic, Produced.with(Serdes.String(), Serdes.String()));
```
Comment (bbejeck) [inline _includes/tutorials/aggregating-sum/kstreams/markup/answer/short-answer.adoc:12]: one final comment - in our short answer version just use `toStream().to(...)` but I'm on the fence about it.  Your call to make the change or not.

**Review by bbejeck — APPROVED** [review]
Thanks for the update @awalther28 just one minor comment, otherwise LGTM

File: _includes/tutorials/aggregating-sum/kstreams/markup/answer/short-answer.adoc (hunk 2 above)
Comment (awalther28) [reply _includes/tutorials/aggregating-sum/kstreams/markup/answer/short-answer.adoc]: Ah thank you for catching that. In commit https://github.com/confluentinc/kafka-tutorials/pull/814/commits/d5f0131fda1a9fc9e03f3ed643803734adefc08d I update the short answer with the " total sales" text.

**Pull request merged**